src/styles/_card.scss

```diff
@@ -0,0 +1,135 @@
+@import 'variables';
+
+.card {
+  width: 200px;
+  background-color: $four-color;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  padding: 32px 16px 16px;
+
+  &__image {
+    display: block;
+    width: 160px;
+    height: 134px;
+    margin: 0 auto;
+    background-repeat: no-repeat;
+    background-size: contain;
+  }
+
+  &__description {
+    width: 166px;
+    height: 36px;
+    font-weight: 500;
+    font-size: $font-size-md;
+    line-height: 18px;
+    color: $triple-color;
+
+    &--margin-top {
+      margin-top: 40px;
+    }
+  }
+
+  &__code {
+    width: 98px;
+    height: 14px;
+    font-weight: 400;
+    font-size: $font-size-sm;
+    line-height: 14px;
+    color: $secondary-color;
+
+    &--margin-top {
+      margin-top: 4px;
+    }
+  }
+
+  &__rating {
+    width: 166px;
+    height: 16px;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+
+    &--margin-top {
+      margin-top: 16px;
+    }
+  }
+
+  &__stars {
+    padding: 0;
+    display: flex;
+    gap: 4px;
+    list-style: none;
+  }
+
+  &__star {
+    width: 16px;
+    height: 16px;
+    background-repeat: no-repeat;
+    background-position: center;
+    background-image: url('../images/star.svg');
+    background-size: contain;
+
+    &--active {
+      background-image: url('../images/star-active.svg');
+    }
+  }
+
+  &__reviews {
+    height: 16px;
+    font-weight: 400;
+    font-size: $font-size-sm;
+    line-height: 20px;
+    color: $triple-color;
+  }
+
+  &__price-block {
+    width: 166px;
+    height: 18px;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+
+    &--margin-top {
+      margin-top: 24px;
+    }
+  }
+
+  &__txt {
+    font-weight: 400;
+    font-size: $font-size-md;
+    line-height: 18px;
+    color: $secondary-color;
+  }
+
+  &__price {
+    font-weight: 700;
+    font-size: $font-size-lg;
+    line-height: 18px;
+    color: $triple-color;
+  }
+
+  &__button {
+    color: $four-color;
+    background-color: $primary-color;
+    border-radius: 5px;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    width: 166px;
+    height: 40px;
+    font-size: 14px;
+    font-weight: 700;
+    line-height: 16px;
+    text-decoration: none;
+    border: 1px solid $primary-color;
+
+    &:hover {
+      color: $primary-color;
+      background-color: $four-color;
```

Comment:
Ensure that the variable `$four-color` is defined in the 'variables' file. If it's not defined, it will cause a compilation error.